MAINTAINERS.yml

```diff
@@ -2705,3 +2705,16 @@ Random:
     - include/zephyr/random/
   labels:
     - "area: Random"
+
+Testing with Renode:
+  status: maintained
+  maintainers:
+    - mateusz-holenko
+  collaborators:
+    - fkokosinski
+  files:
+    - cmake/emu/renode.cmake
+    - boards/*/*/support/*.repl
+    - boards/*/*/support/*.resc
+  labels:
+    - "area: Renode"
```

Comment:
In my opinion it makes sense - while we can be easily added to or follow the build area, adding/following every platform that uses Renode doesn't scale very well, and adding the "Testing with Renode" area would notify us whenever someone makes changes affecting these platforms, allowing as to potentially add our insight about these changes. While currently only 5 platforms are explicitly supported (from Zephyr's perspective), we already support more than 200 platforms in Renodepedia and we plan to gradually contribute their support to the Zephyr repository. Creating an area for testing in Renode would thus help us monitor/maintain this effort.

Reply:
Related discussion on Discord: https://discord.com/channels/720317445772017664/989195595572973648/1011597885076418570

Reply:
As discussed on Discord:

> I think we can do this for now:
1. add a comment saying that "this area is to be converted to a subarea"
2. add him as a collaborator because subareas are not supposed to have maintainers.
the idea is: there will be "Simulation" top-level area with a maintainer, and subareas for QEMU, Renode and so on with collaborators/contributors.

Reply:
I think that is a reasonable solution. @fkokosinski can you add a commit that adds such a comment?

Reply:
I've moved @mateusz-holenko from `maintainers` to `collaborators`, and added the subarea comment in the most recent push